mocks/mock-bip-claims-api/src/main/java/gov/va/vro/mockbipclaims/controller/ClaimsController.java

```diff
@@ -34,4 +37,23 @@ public ResponseEntity<ClaimDetailResponse> getClaimById(Long claimId) {
 
     return new ResponseEntity<>(response, HttpStatus.OK);
   }
+
+  @Override
+  public ResponseEntity<CloseClaimResponse> cancelClaimById(
+      Long claimId, CloseClaimRequest closeClaimRequest) {
```

Comment:
@dfitchett do you have any context on how this `CloseClaimRequest` is used within the cancel claim PUT request? What are they expecting in it?